Header                    header

geometry_msgs/Pose        pose
geometry_msgs/Twist       velocity
geometry_msgs/Twist       acceleration

geometry_msgs/Pose        pose_ref
geometry_msgs/Vector3	  	velocity_ref

float32                   battery_voltage
float32                   battery_percentage
float32                   signal_rssi